src/TextareaAutosize.js

```diff
@@ -98,10 +98,14 @@ export default class TextareaAutosize extends React.Component {
 
 }
 
-let onNextFrame = window.requestAnimationFrame;
+if (process.browser) {
```

Comment:
In case `componentDidMount` will execute in window-less environments (unlikely but probable) that will make the error message confusing.

I moved code which reference `window` to within the `onNextFrame` function so error message will mention `window`. This also prevents previous lint failures due to referencing undefined `onNextFrame`.

I released your changes in `2.1.2`. Thank you.